### DISCUSSION TITLE
Update Environment Variable via cli like vercel

### DISCUSSION THREAD

**Discussion by kidylee** [Dokploy/dokploy#476] [Ideas]
Vercel has a command line to pull environment variables from cloud to a local .env file and could push a local .env file to update environment variables on the cloud.

Maybe we can introduce a cli tool to do not only environment variable but also other management works.

**Comment by b3nab** [comment]
It sounds like a good improvements on DX, maybe it's not the top priority but definitely will help.

Anyway there is already a CLI tool for Dokploy, you can take a look here: https://github.com/Dokploy/cli

**Comment by Siumauricio** [comment]
Yeah someone asked a similar question on discord, make the CLI more usable like uploading the projects from your current workspace, updating enviroment variables, we could improve in the future.
